flask/api/v1/reservations/route.py

```
@@ -1,7 +1,10 @@
 from flask import Blueprint, request, jsonify
-from models import Reservation, Ticket, SeatLock, Seat, Showtime
+from models import Reservations, Tickets, SeatLocks, Seats, Showtimes  # Updated to DDL-first models
+from serializers import ModelSerializer
 from extensions import db
 from datetime import datetime, timedelta
+from sqlalchemy import select
+from sqlalchemy.orm import selectinload
```

Ruff (0.11.9)

`sqlalchemy.orm.selectinload` imported but unused

Remove unused import: `sqlalchemy.orm.selectinload`

(F401)